create_infotexts.py

```diff
@@ -392,18 +396,18 @@ def process_depicted_place(self, place_string, places_mapping, desc_string):
                 else:
                     place_as_wikitext += place_string
 
-                # Don't forget to add the commons categories, even though only wikidata is used in depicted people field
-                if places_mapping[place_string].get('commonscat'):
-                    self.content_cats.append(places_mapping[place_string]["commonscat"])
-
             else:
                 place_as_wikitext += place_string
 
+
         else:
             place_matches = []
             for place in places_mapping:
                 if place.lower() in desc_string.lower():
                     place_matches.append(place)
+                    # Don't forget to add the commons categories if present.
+                    if places_mapping[place].get('commonscat'):
```

Comment:
Don't move this up from [L415](https://github.com/mattiasostmar/SMVK-Cypern_2017-01/pull/35/files#diff-7dcfcee85feab8260c18d5fc3b799639L415). This was on purpose inside the `if len(place_matches) == 1:` to ensure we only add the category on an exact match.

That said I noticed that on the line before [L409](https://github.com/mattiasostmar/SMVK-Cypern_2017-01/pull/35/files#diff-7dcfcee85feab8260c18d5fc3b799639L409) we should have introduced: `place=place_matches[0]`.

The commonscat on [L423](https://github.com/mattiasostmar/SMVK-Cypern_2017-01/pull/35/files#diff-7dcfcee85feab8260c18d5fc3b799639L423) was correctly removed.